### ISSUE TITLE
Accuracy for 10 shots without finetuning

### ISSUE THREAD

**Issue by Sundragon1993** [infinity7428/hANMCL#4]
Dear author,

Thanks so much for your fantastic work, could you publish the accuracy when training 10 shots without fine-tuning? Why don't you report this accuracy in the paper?

**Comment by infinity7428** [comment]
I also revised the paper and added it. However, performance improvement is not a leap forward.

**Comment by Sundragon1993** [comment]
@infinity7428 What does the "Way: category of classification" mean? Is it "Ft" in the earlier version?
